examples/decorator.rs

```diff
@@ -23,8 +23,7 @@ fn format_helper(
     let param = h
         .param(0)
         .ok_or(RenderError::new("Param 0 is required for format helper."))?;
-    let rendered = format!("{} pts", param.value().render());
-    out.write(rendered.as_ref())?;
+    write!(out, "{} pts", param.value().render())?;
```

Comment:
Is that done here?

Reply:
Sorry I just realized `write!` uses `write_fmt` under the hood.